cves/CVE-2011-1183.yml

```diff
@@ -139,7 +149,10 @@ interesting_commits:
     emerging themes?
     If there are no interesting commits, demonstrate that you completed this
     section by explaining what happened between the VCCs and the fix.
-  answer:
+  answer: |
+    The thing with this issue is that it was introduced via an 'incorrect fix' to two 
```

Comment:
I would say to get rid of `"The thing with this issue is that"` part. It feels very informal and in my opinion detracts from the overall statement.

Reply:
Will do this, thanks.